common/src/configuration/bootstrap.rs

```diff
@@ -349,6 +353,7 @@ mod test {
 
     #[test]
     fn check_homedir_is_used_by_default() {
```

Comment:
Can't this little test just be put in the above test rather than adding the risk of a dead lock?